examples/trusted_information_retrieval/module/rust/src/lib.rs

```diff
@@ -74,6 +70,57 @@ impl TrustedInformationRetrievalNode {
         // Compute distance.
         earth_radius * central_angle
     }
+
+    /// Create a gRPC client pseudo-Node.
+    fn get_grpc_client() -> Result<DatabaseClient, String> {
+        match oak::grpc::client::Client::new_with_label(
+            &oak::node_config::grpc_client("https://localhost:8888"),
+            &Label::public_untrusted(),
+        )
+        .map(DatabaseClient)
+        {
+            Some(grpc_client) => Ok(grpc_client),
+            None => Err("Couldn't create a gRPC client".to_string()),
+        }
+    }
+
+    /// Get database size through a gRPC request.
+    fn get_database_size() -> Result<u32, String> {
+        match Self::get_grpc_client()?.get_database_size(GetDatabaseSizeRequest {}) {
+            Ok(response) => Ok(response.size as u32),
+            Err(error) => Err(format!("gRPC send error: {:?}", error)),
+        }
+    }
+
+    /// Load a database subset defined by the `first` and the `last` entry indices (the `last`
+    /// entry is included in the response).
+    fn list_database_entries(first: u32, last: u32) -> Result<Vec<PointOfInterest>, String> {
```

Comment:
Could you make this `first` and `page_size`, so that it is more in line with https://cloud.google.com/apis/design/design_patterns#list_pagination ?

Reply:
Done